docs/development_guide/install_development.rst

```diff
@@ -289,3 +289,11 @@ use and configure it.
 .. _pyramid_debugtoolbar: https://github.com/Pylons/pyramid_debugtoolbar
 .. _pyramid_debugtoolbar documentation: http://docs.pylonsproject.org/projects/pyramid-debugtoolbar/en/latest/
 
+
+Feature Flags
+-------------
+
+Features under development at hypothesis are hidden behind feature flags. You can enable them
```

Comment:
Feature flags aren't just for features currently under development. The idea is that every feature (well, most) is behind a feature flag, all flags are turned off by default, and the site admins can choose which features to have on for which users. I don't think we're gonna remove the groups feature flag once groups is done, for example, we'll just turn the flag on for our production deployment.